sdks/python/src/opik/id_helpers.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+import uuid
+
+
+# STATIC NS
+UUIDV7_NS = 1733244176020523256
+
+
+def uuid4_to_uuid7(user_datetime: datetime, user_uuid: str) -> uuid.UUID:
```

Comment:
Please use the provided datetime. Not a big fan of the hardcoded option, but if there's no way, you can pass it as the default value.